tests/acceptance/features/apiCollaboration/checkFileInfo.feature

```diff
@@ -289,3 +289,302 @@ Feature: check file info with different wopi apps
         }
       }
       """
+
+
+  Scenario Outline: check file info with different mode (onlyOffice)
+    Given user "Alice" has uploaded file with content "hello world" to "/textfile0.txt"
+    When user "Alice" checks the information of file "textfile0.txt" of space "Personal" using office "OnlyOffice" with view mode "<mode>"
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": [
+          "BaseFileName",
+          "Version",
+          "BreadcrumbDocName",
+          "BreadcrumbFolderName",
+          "BreadcrumbFolderUrl",
+          "PostMessageOrigin",
+          "DisablePrint",
+          "UserFriendlyName",
+          "UserId",
+          "ReadOnly",
+          "UserCanNotWriteRelative",
+          "UserCanRename",
+          "UserCanWrite",
+          "SupportsLocks",
+          "SupportsRename",
+          "SupportsUpdate"
+        ],
+        "properties": {
+          "BaseFileName": {
+            "const": "textfile0.txt"
+          },
+          "UserId": {
+            "type": "string"
+          },
+          "Version": {
+            "type": "string"
+          },
+          "SupportsLocks": {
+            "const": true
+          },
+          "SupportsRename": {
+            "const": true
+          },
+          "SupportsUpdate": {
+            "const": true
+          },
+          "UserFriendlyName": {
+            "const": "Alice Hansen"
+          },
+          "ReadOnly": {
+            "const": false
+          },
+          "UserCanNotWriteRelative": {
+            "const": false
+          },
+          "UserCanRename": {
+            "const": <userCanRename>
+          },
+          "UserCanWrite": {
+            "const": <userCanWrite>
+          },
+          "DisablePrint": {
+            "const": <disablePrint>
+          },
+          "BreadcrumbDocName": {
+            "const": "textfile0.txt"
+          },
+          "BreadcrumbFolderName": {
+            "const": "Alice Hansen"
+          },
+          "BreadcrumbFolderUrl": {
+            "type": "string"
+          },
+          "PostMessageOrigin": {
+            "type": "string"
+          }
+        }
+      }
+      """
+    Examples:
+      | mode  | disablePrint | userCanWrite | userCanRename |
```

Comment:
I we are requiring only `mode` as a parameter in the step-definition for this particular step, then is it necessary to use other examples like `disablePrint` and others except `mode` in the `example table`?

Reply:
please look wisely again maybe you find use of other parameter

Reply:
>is it necessary to use other examples like disablePrint and others except mode in the example table?

depending upon the mode, the respective properties are different